src/animation.js

```diff
@@ -12,9 +12,9 @@
 //     See the License for the specific language governing permissions and
 // limitations under the License.
 
-(function(scope, testing) {
+(function(shared, scope, testing) {
 
-  var sequenceNumber = 0;
+  shared.sequenceNumber = 0;
```

Comment:
> why is the sequence number shared?

I kept having to add sequence numbers to web-animations-next-animations for the purpose of debugging, so I decided to add them permanently. Is there a reason why we can't have sequence numbers on WANAnimations and Animations?